.github/instructions/dsc-community-style-guidelines-pester.instructions.md

```
@@ -15,7 +16,7 @@ applyTo: "**/*.[Tt]ests.ps1"
 - Pass all mandatory parameters to avoid prompts
 
 ## Structure & Scope
-- Public commands: Never use `InModuleScope`
+- Public commands: Never use `InModuleScope` (unless retrieving localized strings)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)